### PR TITLE
HADOOP-18214. Update BUILDING.txt

### PR DESCRIPTION
**Pull request by gvieri**
### Description of PR
java-8-openjdk become openjdk-8-jdk (see both ubuntu and debian package's name)

### How was this patch tested?
with simple command issued on command line 

### For code changes:

- [ ] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'HADOOP-17799. Your PR title ...')?
- [ ] Object storage: have the integration tests been executed and the endpoint declared according to the connector-specific documentation?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the `LICENSE`, `LICENSE-binary`, `NOTICE-binary` files?

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 59s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +1 :green_heart: |  @author  |   0m  1s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  shadedclient  |  28m 15s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  shadedclient  |  21m 44s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 30s |  |  The patch does not generate ASF License warnings.  |
|  |   |  53m 10s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3811/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/3811 |
| Optional Tests | dupname asflicense codespell |
| uname | Linux 5c1c456181a2 4.15.0-163-generic #171-Ubuntu SMP Fri Nov 5 11:55:11 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 9c313e060eea4cd16eaa8358c29422bb977c7a5e |
| Max. process+thread count | 571 (vs. ulimit of 5500) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-3811/1/console |
| versions | git=2.25.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

**Review by ayushtkn — APPROVED** [review]
Couldn't actually try on a ubuntu box. But found a page, which mentions the same command 
https://docs.datastax.com/en/jdk-install/doc/jdk-install/installOpenJdkDeb.html

File: BUILDING.txt [hunk 1]
```diff
@@ -57,7 +57,7 @@ Refer to  dev-support/docker/Dockerfile):
 
 * Open JDK 1.8
   $ sudo apt-get update
-  $ sudo apt-get -y install java-8-openjdk
+  $ sudo apt-get -y install openjdk-8-jdk
```
Comment (jojochuang) [inline BUILDING.txt:60]: this is in fact the package we use in the Dockerfile build script. https://github.com/apache/hadoop/blob/trunk/dev-support/docker/pkg-resolver/packages.json#L264

**Review by jojochuang — APPROVED** [review]

**Comment by jojochuang** [comment]
I merged this PR. But it would be nice to create an Apache JIRA next time.

**Comment by steveloughran** [comment]
I will create that jira before i cherrypick to branch3.3

**Comment by steveloughran** [comment]
created https://issues.apache.org/jira/browse/HADOOP-18214 for this; cherrypicked with that jira in the title to branch 3.3.3 and 3.3

**Comment by aajisaka** [comment]
Hi @gvieri, thank you for your contribution!

Could you let me know your ASF JIRA account? I want to set you as the assignee of https://issues.apache.org/jira/browse/HADOOP-18214

**Comment by gvieri** [comment]
It is necessary to have ASF JIRA account ?

**Comment by ayushtkn** [comment]
@gvieri well yes. This one got committed initially without a Jira since it wasn’t changing the core code, but in general we tend to have jira for almost everything. That’s how we track issues.

BTW. If you have a Jira account and have it assigned on your name. You will get the credit in the Release Notes as well when Hadoop does the release. Just one of the things if that interests or motivates you :-)

**Pull request merged**